experimental/traffic-portal/src/app/core/users/roles/detail/role-detail.component.scss

```diff
@@ -0,0 +1,30 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+mat-card {
+	margin: 1em auto;
+	width: 80%;
+	min-width: 350px;
+
+	mat-card-content {
+		display: grid;
+		grid-template-columns: 1fr;
+		row-gap: 2em;
+		margin: 1em auto 50px;
+		textarea{
+			white-space: pre-wrap;
+			max-height: 250px;
+		}
+	}
+}
```

Comment:
Instead of repeating this here, we should make use of the `src/app/core/styles/form.page.scss` stylesheet in the component's `stylesheetUrls` decorator property.